packages/fondue/src/components/RichTextEditor/Plugins/EmojiPlugin/EmojiPicker/EmojiPickerSearchAndClear.tsx

```diff
@@ -0,0 +1,30 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Icons } from '@components/RichTextEditor/components/Toolbar/Icons';
+import { cn } from '@udecode/cn';
+import { UseEmojiPickerType } from '@udecode/plate-emoji';
+
+export type EmojiPickerSearchAndClearProps = Pick<UseEmojiPickerType, 'i18n' | 'searchValue' | 'clearSearch'>;
+
+export function EmojiPickerSearchAndClear({ i18n, searchValue, clearSearch }: EmojiPickerSearchAndClearProps) {
```

Comment:
Inline adding an emoji doesn't work with they keyboard.

For example:
- type ":hear"
- press "ArrowDown" to select heart emoji
- Press "Enter"
- No selection is added to content

Reply:
Fixed!